src/listTask.js

```diff
@@ -0,0 +1,60 @@
+import Tasks from './Tasks.js';
+
+const displayAllTasks = () => {
```

Comment:
This is a very big function and could perhaps be a little difficult for another person to read or debug, do you think it might be possible to split into multiple smaller functions?